deps/exokit-bindings/canvascontext/src/canvas-context.cc

```diff
@@ -426,25 +460,126 @@ NAN_SETTER(CanvasRenderingContext2D::StrokeStyleSetter) {
     std::string strokeStyle(*text, text.length());
 
     canvas::web_color webColor = canvas::web_color::from_string(strokeStyle.c_str());
-    context->strokePaint.setColor(((uint32_t)webColor.a << (8 * 3)) | ((uint32_t)webColor.r << (8 * 2)) | ((uint32_t)webColor.g << (8 * 1)) | ((uint32_t)webColor.b << (8 * 0)));
-    context->fillPaint.setShader(nullptr);
+    uint32_t rgba = ((uint32_t)webColor.a << (8 * 3)) | ((uint32_t)webColor.r << (8 * 2)) | ((uint32_t)webColor.g << (8 * 1)) | ((uint32_t)webColor.b << (8 * 0));
+    context->strokePaint.setColor(rgba);
+    context->strokePaint.setShader(nullptr);
+    rgbaToString(strokeStyle, rgba);
+    context->jsStrokeStyle.Reset(JS_STR(strokeStyle.c_str()));
   } else if (value->IsObject() && JS_OBJ(JS_OBJ(value)->Get(JS_STR("constructor")))->Get(JS_STR("name"))->StrictEquals(JS_STR("CanvasGradient"))) {
     CanvasRenderingContext2D *context = ObjectWrap::Unwrap<CanvasRenderingContext2D>(info.This());
 
     CanvasGradient *canvasGradient = ObjectWrap::Unwrap<CanvasGradient>(Local<Object>::Cast(value));
     context->fillPaint.setShader(canvasGradient->getShader());
+    context->jsStrokeStyle.Reset(value);
   } else if (value->IsObject() && JS_OBJ(JS_OBJ(value)->Get(JS_STR("constructor")))->Get(JS_STR("name"))->StrictEquals(JS_STR("CanvasPattern"))) {
     CanvasRenderingContext2D *context = ObjectWrap::Unwrap<CanvasRenderingContext2D>(info.This());
 
     CanvasPattern *canvasPattern = ObjectWrap::Unwrap<CanvasPattern>(Local<Object>::Cast(value));
     context->fillPaint.setShader(canvasPattern->getShader());
+    context->jsStrokeStyle.Reset(value);
   } else {
     Nan::ThrowError("strokeStyle: invalid arguments");
   }
 }
 
+void CanvasRenderingContext2D::Configure() {
```

Comment:
I don't think this is an appropriate name for this method. It's not really configuring anything, and strictly deals with shadows.

Maybe a local function (not method) like `makeDrawLooper` would be more appropriate?

Reply:
It could be a local function, but then the member variables would need to stay public. I see that `protect:` is commented out in the header.

(I agree about the name though.)